test/setup-kubernetes.sh

```diff
@@ -61,8 +61,9 @@ networking:
   podSubnet: \"10.244.0.0/16\""
 	;;
     fedora)
-	# Use weave on Fedora. Nothing to add to kubeconfig.
-	podnetworkingurl=https://cloud.weave.works/k8s/net?k8s-version=$k8sversion
+	# Use Calico on Fedora as it is the only CNI plugin that is tested by kubeadm.
+	# https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network
+	podnetworkingurl=https://docs.projectcalico.org/manifests/calico.yaml
```

Comment:
Can we lock down the URL to a specific version?

We didn't do that for weave either and as the regression in our CI shows, this is potentially problematic - if the content of that download changes, we cannot be sure that it continues working.

Reply:
Seems like there is no version specific manifests for calico. It also has an operator for installing and managing the plugin, but it's not suitable for out case.
Ref: https://docs.projectcalico.org/getting-started/kubernetes/self-managed-onprem/onpremises